### ISSUE TITLE
The current notebook creation process in vs studio code is not user-friendly

### ISSUE THREAD

**Issue by smpa01** [microsoft/SynapseVSCode#42]
Whenever I create a new workbook locally or download an existing fabric notebook in synapse, it completely ignores any workspace already opened in vs studio code and opens the new notebook workspace in vscode. 

This is extremely inconvenient, because I want to keep working on my current workspace, without the change of context. Is there a way to work around this? In my opinion, the notebook directory should be added to the current workspace, or the action should be cancelled altogether if the current workspace already includes the notebook directory (for example, the parent directory of the notebook directory is already open in vscode).

**Comment by jtcrd** [comment]
Agree, it is a pain that we can't navigate back to the original workspace
